src/components/MultipleAvatars.js

```diff
@@ -63,7 +64,7 @@ const MultipleAvatars = ({
                         />
                     ) : (
                         <View
-                            style={singleAvatarStyles}
+                            style={[singleAvatarStyles, styles.alignItemsCenter, styles.justifyContentCenter]}
                         >
                             <Text style={size === 'small'
                                 ? styles.avatarInnerTextSmall
```

Comment:
@parasharrajat I thought as part of the fix for this you were going to remove the custom line-height based styles?

Reply:
Yes. I did planned this earlier but I realized that the font-family for the badge is not correct as we are using Text from RN instead of our customized version. But the new customized `Text` impose line-height of 20 which I think could be bigger for the LHN Focused mode thus keeping the line-height is a safe move. I am not sure how to reset the Line-height here as we don't actually need it.

Reply:
> thus keeping the line-height is a safe move

Hmmm ... I'm confused how you landed here. Since our custom `Text` component specifies a line-height, we should eliminate any unnecessary external styles which might conflict with the one specified internally.

As an aside (not to be handled in this PR, I think we should rename our custom `Text` component to `Expensitext`, to avoid the common mistake of importing `Text` from RN instead, which should never be done in Expensify.cash AFAIK.

Reply:
> Since our custom Text component specifies a line-height, we should eliminate any unnecessary external styles which might conflict with the one specified internally
Normally, it have 20 LH which is fine. But when we are on #focused mode LH 20 is bigger for the avatar. it should be 18.

**Yeah let me check it again if LH is something we need to worry at all and then I will remove those from styles**.

There are many places where we are wrongly using the RN's `Text` I will send another PR for this. May be we can look at renaming it there.

Reply:
@roryabraham It seems the extra Lineheight:20 is problematic in #focus mode. So either we keep these line heights or reset our Text's lH using `lineHeight:undefined` instead of these LHs.

Reply:
@roryabraham I updated the PR with `lineHeight:undefined` as I found it more promising.Thanks.